instrumentation/github.com/aws/aws-sdk-go/service/otels3/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
This doesn't match our [naming standards for instrumentation](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/master/instrumentation#packaging). A parent directory of `s3` is needed, making the full import path of this package `go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-sdk-go/service/s3/otels3`